nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,425 @@
+        from fastapi.responses import JSONResponse
+
+        secret = request.headers.get('X-Telegram-Bot-Api-Secret-Token', '')
+        if secret != self._webhook_secret:
+            return JSONResponse({'ok': False}, status_code=403)
+
+        try:
+            update = await request.json()
+            task = asyncio.create_task(self._handle_update(update))
+            self._inflight.add(task)
+            task.add_done_callback(self._inflight.discard)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)